src/index.js

```diff
@@ -71,158 +71,188 @@ class App extends React.Component {
-           />
-         </label>
-         <br/>
-         <label>Optional Transcript Name</label>
-         <input
-           type="text"
-           value={ this.state.title }
-           onChange={ e => this.handleChangeTranscriptTitle(e.target.value) }
-         />
-
-         <br />
-         <button onClick={ () => this.clearLocalStorage() }>Clear Local Storage</button>
-         <hr/>
-
-         <TranscriptEditor
-           transcriptData={ this.state.transcriptData }
-           fileName={ this.state.fileName }
-           mediaUrl={ this.state.mediaUrl }
-           isEditable={ this.state.isTextEditable }
-           sttJsonType={ this.state.sttType }
-           handleAnalyticsEvents={ this.handleAnalyticsEvents }
-           title={ this.state.title }
-           ref={ this.transcriptEditorRef }
-         />
-         <hr/>
-         <label>Components Analytics</label>
-         <textarea
-           style={ { height: '200px', width: '100%' } }
-           value={ JSON.stringify(this.state.analyticsEvents, null, 2) }
-           disabled>
-         </textarea>
-       </div>
-     );
-   }
+          </a>
+          <hr />
+        </span>
+
+        <div className={ style.demoNav }>
+          <section className={ style.demoNavItem }>
+            <button onClick={ () => this.loadDemo() }>load demo</button>
+          </section>
+          <section className={ style.demoNavItem }>
+            <label>Load Local Media</label>
+            <br />
+            <input
+              className={ style.demoInput }
+              type="file"
+              onChange={ e => this.handleChangeLoadMedia(e.target.files) }
+            />
+            <br />
+            or
+            <br />
+            <button onClick={ () => this.handleChangeLoadMediaUrl() }>
+              Load Media From Url
+            </button>
+          </section>
+          <section className={ style.demoNavItem }>
+            <label>
+              open Transcript <code>Json</code>
+            </label>
+            <br />
+            <SttTypeSelect
+              name={ 'sttType' }
+              value={ this.state.sttType }
+              handleChange={ this.handleSttTypeChange }
+            />
+            <br />
+            <input
+              className={ style.demoInput }
+              type="file"
+              onChange={ e =>
+                this.handleChangeLoadTranscriptJson(e.target.files)
+              }
+            />
+          </section>
+          <section className={ style.demoNavItem }>
+            <label>Export transcript</label>
+            <br />
+            <ExportFormatSelect
+              name={ 'exportFormat' }
+              value={ this.state.exportFormat }
+              handleChange={ this.handleExportFormatChange }
+            />
+            <br />
+            <button onClick={ () => this.exportTranscript() }>Export file</button>
+          </section>
+          <section className={ style.demoNavItem }>
```

Comment:
do we still need text is editable?!

Reply:
yeah, I'd still keep Text Is Editable, eg think situations like the BBC Volt Use case, where v1 is not editable and v2 is. Or there might be a situation where you have an admin user that edits the text and another user that can only view it but not edit etc.. Or we do another prototype, different use case and you want to have the "interactive transcript" but non editable (eg is a new story format) etc..